event-app-api/server.js

```diff
@@ -48,43 +82,51 @@ app.post("/signup", (req, res) => {
 
   bcrypt.hash(password, saltRounds, (err, hash) => {
     if (err) {
-      console.log(err);
+      res.send(err);
     }
+    //used the sequelize.query method to insert data into the Users table
     sequelize.query(
       `INSERT INTO "public"."Users" (fullname, username, password, email, tel, "createdAt", "updatedAt") VALUES ('${fullname}', '${username}', '${hash}', '${email}', '${tel}', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP)`,
       { type: sequelize.QueryTypes.INSERT },
       (err, result) => {
-        console.log(err);
+        res.send(err);
       }
     );
   });
 });
 
+//setting a POST request to the login endpoint
 app.post("/login", async (req, res) => {
   const username = req.body.username;
   const password = req.body.password;
-
+  //Used the sequelize.query method to retrieve data from the user's table
   const result = await sequelize.query(
     `SELECT * FROM "public"."Users" WHERE username = '${username}'`,
     { type: sequelize.QueryTypes.SELECT }
```

Comment:
Earlier in your /profile endpoint you have this to query the user:
```
const profileInfo = await User.findOne({
    attributes: { exclude: ["password"] },
    where: { username: username },
  });
```
Is querying it this way more optimal?